src/contentmanagermodel.cpp

```diff
@@ -61,14 +61,17 @@ QVariant ContentManagerModel::data(const QModelIndex& index, int role) const
 
     const auto col = index.column();
     const bool isThumbnailRequest = col == 0 && role == Qt::DecorationRole;
-    const bool otherDataRequest = col != 0 && (role == Qt::DisplayRole || role == Qt::UserRole+1 );
+    const bool isDescriptionRequest =
+        col == 0 && (role == Qt::DecorationRole || role == Qt::DisplayRole) &&
+        index.parent().isValid();
```

Comment:
1. Where do you think is the best place to put isDescriptionLine(index)? I personally think descriptionnode.h is the best place as placing a view, model, or the contentmanager class into delegate somewhat doesn't make sense given a MVC architecture.
2. It was previously needed because the isThumbNail overlaps this. Now that I think about it I can use isDescriptionLine to distinguish between them instead of role.
```
const bool isThumbnailRequest = col == 0 && role == Qt::DecorationRole;
...
if ( !isThumbnailRequest && !otherDataRequest && !isDescriptionRequest)
        return QVariant();
```

Reply:
> 1. Where do you think is the best place to put isDescriptionLine(index)?

Please define it in `contentmanagermodel.h` as a free function

Reply:
Done